websub-ballerina/src/websub/natives.bal

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
These files are not newly added. These are migrated from here(https://github.com/ballerina-platform/ballerina-lang/tree/v2.0.0-Preview1/stdlib/websub).  We only reimplement the tests.